### PR TITLE
Autosave by observing model.scratch in editor

### PR DESCRIPTION
**Pull request by novaugust**
No issue
- removes keypress handling in the editor component
- automated value changes via shortcuts still autosave

### CODE REVIEW

File: core/client/app/mixins/editor-base-controller.js [hunk 1]
```diff
@@ -32,6 +32,21 @@ EditorControllerMixin = Ember.Mixin.create({
         };
     },
 
+    scratch: null,
+    autoSave: function () {
+        console.log("Foo");
```
Comment (ErisDS) [inline core/client/app/mixins/editor-base-controller.js]: :penguin:

**Comment by ErisDS** [comment]
Whops, I've spotted an unintended side effect here:

In the case that you start a new post, the autosave fires after 3 seconds, regardless of whether you have focused into the editor and typed something. So if you're in the middle of typing a really long title, the screen suddenly redraws and some of your text lands in the editor.

Also seeing this error, but not sure if it's due to this PR or may also occur on master, will check

**Comment by novaugust** [comment]
Ah, sounds like something on this PR. I'll get you more info in a bit

**Comment by novaugust** [comment]
Updated with a new commit - will squash if the new code's worth it

**Comment by novaugust** [comment]
This is all shiny and hopefully GTG

**Comment by ErisDS** [comment]
Taken this for a spin and it seems like it's working and quite a lot simpler :+1:

**Pull request merged**